spicedb/check_docker_podman.sh

```diff
@@ -19,12 +19,18 @@ else
     exit 1
   fi
 
-   # Check if Podman is installed
+   # Check if docker-compose is installed
     if command_exists docker-compose; then
       echo "docker-compose is installed."
     else
-      echo "docker-compose not found. Please install either docker-compose or podman-compose to proceed."
-      exit 1
+
```

Comment:
This update will only echo only. 
Script: https://github.com/project-kessel/relations-api/blob/main/spicedb/start-insights-rebac.sh#L6
continue to use docker compose as the default compose tool.

Newer version of docker-compose follows: `docker compose` vs `docker-compose`. I think Will had issue on linux for docker-compose.

Reply:
Add `export COMPOSE=docker-compose` or  `export COMPOSE=podman-compose`

Update the https://github.com/project-kessel/relations-api/blob/main/spicedb/start-insights-rebac.sh#L6 to use the COMPOSE var

Reply:
yes, remove them would make sense.